src/sdk/pynni/nni/__main__.py

```diff
@@ -27,28 +27,40 @@
 import json
 import importlib
 
+from .constants import ModuleName, ClassName, ClassArgs
 from nni.msg_dispatcher import MsgDispatcher
-from nni.hyperopt_tuner.hyperopt_tuner import HyperoptTuner
-from nni.evolution_tuner.evolution_tuner import EvolutionTuner
-from nni.batch_tuner.batch_tuner import BatchTuner
-from nni.medianstop_assessor.medianstop_assessor import MedianstopAssessor
 
 logger = logging.getLogger('nni.main')
 logger.debug('START')
 
-BUILT_IN_CLASS_NAMES = ['HyperoptTuner', 'EvolutionTuner', 'BatchTuner', 'MedianstopAssessor']
+def augment_classargs(input_class_args, classname):
+    if classname in ClassArgs:
+        for key, value in ClassArgs[classname].items():
+            if key not in input_class_args:
+                input_class_args[key] = value
+    return input_class_args
 
 def create_builtin_class_instance(classname, jsonstr_args):
+    if classname not in ModuleName or \
+        importlib.util.find_spec(ModuleName[classname]) is None:
+        raise RuntimeError('Tuner module is not found: {}'.format(classname))
+    class_module = importlib.import_module(ModuleName[classname])
+    class_constructor = getattr(class_module, ClassName[classname])
     if jsonstr_args:
         class_args = json.loads(jsonstr_args)
-        instance = eval(classname)(**class_args)
+        class_args = augment_classargs(class_args, classname)
```

Comment:
Suggest to merge line 50 and 51 into one line, like:
class_args = augment_classargs(json.loads(jsonstr_args), classname)

Reply:
done